src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -1935,8 +1935,9 @@ namespace Microsoft.FSharp.Collections
         /// <param name="rank">The rank of the index.</param>
         /// <param name="offset">The offset from the end.</param>
         /// <returns>The corresponding index from the start.</returns>
+        [<Experimental("Experimental library feature, requires '--langversion:preview'")>]
```

Comment:
Could we make this a constant and only internal to FSharp.Core?
```fsharp
[<Literal>]
val internal PreviewLibraryFeatureMarker : string = "Experimental library feature, requires '--langversion:preview'"
```
"Experimental library feature, requires '--langversion:preview'"